README.md

```diff
@@ -409,26 +431,31 @@ sudo ./check-production.sh
 		Apr 09 22:54:50 right-glider-515046 systemd[1]: Started mpa_registration_tool.service - Intel MPA Registratio>
 		Apr 09 22:54:50 right-glider-515046 systemd[1]: mpa_registration_tool.service: Deactivated successfully.
 		```
-	2. Check MPA log file with following command:
+
+	2. Check the MPA log file.
 		```bash
 		cat /var/log/mpa_registration.log
 		```
 
-		A successful example output:
+		A example output of a successful registration:
 
-		```bash
```

Comment:
Undo change. This is still a result from bash

Reply:
i'm changed to ```console